packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -307,10 +289,11 @@ contract VotingV2 is
     constructor(
         uint256 _emissionRate,
         uint256 _spamDeletionProposalBond,
-        uint64 _unstakeCoolDown,
-        uint64 _phaseLength,
-        uint64 _minRollToNextRoundLength,
+        uint256 _unstakeCoolDown,
+        uint256 _phaseLength,
+        uint256 _minRollToNextRoundLength,
         uint256 _gat,
+        uint256 _startingRequestIndex,
```

Comment:
nit: we should probably make this uint64 since we assume that starting request index is within that range through most of the app.